.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: On release, build contracts
```

Comment:
Would be useful to add this workflow, as well - at least, to ensure reproducible builds work:

https://github.com/multiversx/mx-reproducible-contract-build-example-sc/blob/main/.github/workflows/on_pull_request_build_contracts.yml